src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -91,6 +91,20 @@ enum class ConfigType : uint16_t {
    */
   HARDWARE_TICK_CONFIG = 22,
 
+  /**
+   * A bitmask indicating which GNSS constellations are enabled.
+   *
+   * Payload format: `uint32_t` (see @ref sat_type_masks)
+   */
+  ENABLED_GNSS_SYSTEMS = 50,
+
+  /**
+   * A bitmask indicating which GNSS frequency bands are enabled.
+   *
+   * Payload format: `uint16_t` (see @ref freq_band_masks)
```

Comment:
Why is this a `uint16_t` while `sat_type_masks` is a `uint32_t`? Seems like it would make more sense to have them be the same type. Ideally, since we generally use 4 byte alignment, I would assume they'd both be `uint32_t`.

Reply:
I did `uint16_t` just because realistically there are not a lot of bands so 16 is more than enough, but you're right that there's no need to limit it here and might be better to consider alignment. I'll change to `uint32_t` instead.